mermaid/arch/examples_test.go

```
@@ -0,0 +1,118 @@
+			arch.Edge{
+				ServiceID: "bottom_gateway",
+				Position:  arch.PositionTop,
+				Arrow:     arch.ArrowNone,
+			},
+			arch.Edge{
+				ServiceID: "junctionRight",
+				Position:  arch.PositionBottom,
+				Arrow:     arch.ArrowNone,
+			}).String() //nolint
+
+	markdown.NewMarkdown(os.Stdout).
```

golangci-lint (1.62.2)

[low] G104: Errors unhandled.

(gosec)